spec/classes/puppet_init_spec.rb

```diff
@@ -52,7 +52,10 @@
       end
 
       describe 'with no custom parameters' do
+        it { is_expected.to compile.with_all_deps }
```

Comment:
I think when running in a Unix environment, the path's being tested with Unix file path expectations too, because that's how Puppet's being run. I've excluded the test from Windows for now.